pipelines/streaming/README.md

```diff
@@ -44,11 +49,11 @@ planned.
     to match your environment. See the documentation on
     [Debezium MySQL Connector properties](https://debezium.io/documentation/reference/connectors/mysql.html#mysql-property-name)
     for more information.
-1.  Build binaries with `mvn clean install`.
+1.  Build binaries from the repo root with `mvn clean package`.
 1.  Run the pipeline to a FHIR server and Parquet files:
 
     ```shell
-    $ java -cp ./pipelines/streaming/target/streaming-bundled-0.1.0-SNAPSHOT.jar \
+    $ java -jar ./pipelines/streaming/target/streaming-bundled-0.1.0-SNAPSHOT.jar \
```

Comment:
I think in these case too you can drop the main class name, i.e., `org.openmrs.analytics.Runner` but please test it before updating. If you don't want to do the test now (given this is in `streaming`) please keep the old tested version.

Reply:
After testing, neither this version or the original version seem to work:

```
Exception in thread "main" com.beust.jcommander.ParameterException: Was passed main parameter '--fhirDatabaseConfigPath=./utils/dbz_event_to_fhir_config.json' but no main parameter was defined in your arg class
```

I'll submit it as is, but I'm not sure why streaming isn't working.